src/pages/SearchPage.js

```diff
@@ -159,31 +161,32 @@ class SearchPage extends Component {
             this.state.searchValue,
         );
         return (
-            <ScreenWrapper>
-                {({didScreenTransitionEnd}) => (
-                    <>
-                        <HeaderWithCloseButton
-                            title={this.props.translate('common.search')}
-                            onCloseButtonPress={() => Navigation.dismissModal(true)}
+            <ScreenWrapper
+                onTransitionEnd={() => {
+                    const searchValue = KeyboardShortcut.consumeBuffer();
+                    this.setState({didScreenTransitionEnd: true, searchValue}, this.updateOptions);
```

Comment:
I think this looks good. 👍 It's cool to see it in action.

Reply:
This does not only put value in the input. It keeps the searched list ready to be shown to the user based on his input. So he does not have to wait for the search to happen.